### PR TITLE
fix(deps): Update to Kotlin 1.8 + Awala dependencies

### PR DESCRIPTION
**Pull request by sdsantos**
Did some simple QA, planning to do more thorough QA with the 3 apps updated.

### CODE REVIEW

**Comment by sdsantos** [comment]
@gnarea instrumented tests are failing on the API 24 emulator, but I've run them on my own emulators API 23 and API 24 and then all pass 🤷

**Comment by gnarea** [comment]
@sdsantos, any chance those 4 failures could point to a real/production issue on Android 7? If not, we can replace the following with a virtual API 24device that works:

https://github.com/relaycorp/relaynet-courier-android/blob/8a09bf57f3f05566ef872f384e8c712e13d683b6/app/firebase-test-lab.yml#L7-L11

**Review by gnarea — DISMISSED** [review]
Thanks! Looks like CI is passing again

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    package="tech.relaycorp.courier">
```
[the remark this answers is not in the document]
Comment (sdsantos) [reply app/src/main/AndroidManifest.xml:3]: Now that is only set on the `app/build.gradle`. But this was an automatic change when I upgraded gradle build tools 😄 .

**Review by gnarea — APPROVED** [review]

**Pull request merged**